pkg/ride/runtime.go

```diff
@@ -367,4 +367,5 @@ type environment interface {
 	rideV6Activated() bool
 	internalPaymentsValidationHeight() uint64
 	maxDataEntriesSize() int
+	protobufTransaction() bool
```

Comment:
we'd better call this method `isProtobufTransaction`

Reply:
Yep, I agree with you. But as you can see `EvaluationEnvironment` struct has `isProtobufTransaction` field. So, I can't call this method as you suggest.

Reply:
isProtobufTx

Reply:
Fixed